charts/dataingestion-service/templates/pv-ebs.yaml

```diff
@@ -0,0 +1,9 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-es
```

Comment:
This name needs to be unique in the cluster, please change it to a unique name or make a reference here and anywhere this is called.

FYI there is a mismatch between this and the PVC which consumes this resource. To match simply change the name here to: {{ .Values.pvc.diPvClaim.storageClass | quote }}